spaceship/lib/spaceship/client.rb

```diff
@@ -595,6 +595,22 @@ def self.spaceship_session_env
       ENV["FASTLANE_SESSION"] || ENV["SPACESHIP_SESSION"]
     end
 
+    # Get contract messages from App Store Connect's "olympus" endpoint
+    def fetch_program_license_agreement_messages
+      all_messages = []
+
+      messages_request = request(:get, "https://olympus.itunes.apple.com/v1/contractMessages")
+      body = messages_request.body
+      if body
```

Comment:
Small thing, instead of having an `if` block here, you can do

```
return all_messages unless body

...
```

Reply:
@KrauseFx, thanks for the suggestion! I am personally against the `unless` operator in Ruby, but as the maintainer of this project, if you feel you'd prefer `unless`, I will change it. Just let me know!

Reply:
You could also do...
```rb
return all_messages if body.nil?
```

Either works but I think what @KrauseFx was going for was an early return if no body which makes the code a bit flatter and easier to read 🙃 

But what you have here matches with the other implementations already in this file so what you got is 👌